src/main/app/claims/models/response/fullDefenceResponse.ts

```diff
@@ -9,14 +10,16 @@ export interface FullDefenceResponse extends ResponseCommon {
   responseType: ResponseType.FULL_DEFENCE
   defenceType: DefenceType
   defence: string
+  whenDidYouPay: WhenDidYouPay
 }
 
 export namespace FullDefenceResponse {
   export function deserialize (input: any): FullDefenceResponse {
     return {
       ...ResponseCommon.deserialize(input),
       defenceType: input.defenceType,
-      defence: input.defence
+      defence: input.defence,
+      whenDidYouPay: input.whenDidYouPay
```

Comment:
That has no chance to work. You should deserialise it properly by calling `WhenDidYouPay#deserialise` method.